src/method/method_query_handler.cpp

```diff
@@ -743,7 +570,7 @@ namespace cubmethod
 	db_session_set_xasl_cache_pinned (m_session, true, false);
       }
 
-    char stmt_type;
+    char stmt_type = CUBRID_STMT_NONE;
```

Comment:
it looks not necessary. the stmt_type is always set the folllowing code sequences.

Reply:
It is to suppress uninitialized variable warning.

Reply:
Ok.